target/linux/ath79/dts/qca9531_comfast_cf-e560ac.dts

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "comfast,cf-e560ac", "qca,qca9531";
+	model = "COMFAST CF-E560AC";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &system;
+		led-failsafe = &system;
+		led-upgrade = &system;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+                pinctrl-names = "default";
+                pinctrl-0 = <&jtag_disable_pins>;
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wan {
+			label = "cf-e560ac:blue:wan";
+			gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
+		};
+
+		lan1 {
+			label = "cf-e560ac:blue:lan1";
+			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
+		};
+
+		lan2 {
+			label = "cf-e560ac:blue:lan2";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+
+		lan3 {
+			label = "cf-e560ac:blue:lan3";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+		};
+
+		lan4 {
+			label = "cf-e560ac:blue:lan4";
+			gpios = <&gpio 11 GPIO_ACTIVE_LOW>;
+		};
+
+		system: net {
```

Comment:
system or net?

Reply:
this is a matter of confusion from me looking at the vendor implementation directly; they call it net, but it appears to be an on-board LED that isn't exposed outside the case.  The power LED is not controllable.

Reply:
Well, then either decide that the vendor is right and call it net consistently, so:
`led_net: net {`

or decide that it's a system LED and the vendor did a mistake, then
`led_system: system {`
and also update the label property.